### DISCUSSION TITLE
我使用mac mini设备用作服务器，安装后无法播放视频

### DISCUSSION THREAD

**Discussion by mengyuefeitian** [jellyfin/jellyfin#10806] [Q&A]
我在一台M2芯片的mac mini主机上安装了jellyfin，最早是10.8.12，最近升级了10.8.13，我发现ffmpeg路径不可编辑了。
由于mac os的jellyfin安装后，默认不包含ffmpeg，所以我是从ffmpeg官网下载后放在/Users/xxx/.local/share/jellyfin/ffmpeg/ffmpeg来使用，之前可以播放正常。升级到10.8.13版本后就开始不能再播放视频了。
![image](https://github.com/jellyfin/jellyfin/assets/86092211/c714af43-a60e-493e-964f-085f03f5a98b)
报错如上图所示：该客户端与媒体不兼容，服务器未发送兼容的媒体格式。
所以导致我现在无法观看视频了，我通过网上查找资料，在应用程序目录也放了ffmpeg，在/usr/local/bin目录都放置了ffmpeg文件，但是目前仍然无法再解码播放视频了。
以下是日志：
[Log3.txt](https://github.com/jellyfin/jellyfin/files/13827325/Log3.txt)
[Log2.txt](https://github.com/jellyfin/jellyfin/files/13827326/Log2.txt)
[Log.txt](https://github.com/jellyfin/jellyfin/files/13827327/Log.txt)

**Comment by wei98k** [comment]
从10.8.13开始就不允许在页面上修改ffmpeg的路径了需要手动在配置文件中修改，macOS的配置文件路径  ~/.config/jellyfin/encoding.xml
修改配置内容 ` "<EncoderAppPath>"和"<EncoderAppPathDisplay>"`  https://jellyfin.org/posts/jellyfin-security-and-you/

**Reply by mengyuefeitian** [reply]
我实际上是再10.8.12版本可以修改路径的时候添加的，尽管升级了10.8.13不能修改配置，但是原来12版本修改的配置还在，只是我不知道是否还在生效。
我的配置路径中只有ffmpeg文件还没有其他文件，现在我在文件夹中加入了FFprobe和FFplay文件后，播放失败的问题偶尔还会遇到但是已经很少了。
我将再次尝试通过配置文件修改使其生效。再次看看效果
